tables/heb.tbl

```diff
@@ -0,0 +1,260 @@
+#https://en.wikipedia.org/wiki/Hebrew_Braille#:~:text=The%20International%20Hebrew%20Braille%20Code
+#also see: 
+#https://huc.edu/library_blog/learning-about-hebrew-braille/#:~:text=The%20Code%20created%20by%20Brevis,books%20written%20in%20Hebrew%20Braille.
+
+#This braille code was developed by a group of both blind and sighted scholars in an academic setting.
+#-maintainers: Paul Geoghegan <contact@envisionly.tech> and Matityhau Yeshurun <yeatersink@gmail.com>
+letter שְּׁ 146-3
+letter שְּׂ 5-156-3
+letter פְּ 5-1234-3
+letter תְּ 2345-3
+letter לְּ 5-123-3
+letter יְּ 5-245-3
+letter מְּ 5-134-3
+letter נְּ 5-1345-3
+letter ּוְ 136-3
+letter דְּ 5-145-3
+letter קְּ 5-12345-3
+letter שְׁ 146-3
+letter בְּ 12-3
+letter שּׂ 5-156
+letter שְׂ 156-3
+letter שּׁ 146-3
+letter כְּ 13-3
+letter בְּ 12-3
+letter כּ 13
+letter לּ 5-123
+letter יּ 5-245
+letter מּ 5-134
+letter ִי 35
+letter נּ 5-1345
+letter ֵי 3456
+letter ּו 136
+letter וּ 136
+letter וֹ 246
+letter פּ 5-1234
+letter הּ 125
+letter קּ 5-12345
+letter שׁ 146
+letter דּ 5-145
+letter בּ 12
+letter שׂ 156
+letter כְּ 13-3
+letter תְּ 5-1456-3
+letter תּ 2345
+letter ✙ 23456
+letter ❑ 12345
+letter ✕ 1235
+letter ✔ 12356
+letter ❦ 126-126-135-156-1234-345-0-126-34-135-156-1234-345
+letter ❥ 345-126-135-156-1234-345-0-126-34-135-156-1234-345
+letter ✘ 35
+letter ❁ 123-1346-1346
+letter ❚ 14
+letter ❘ 1256
+letter ❍ 135
+letter ❏ 135
+letter ❐ 1235
+letter ❙ 2345
+letter ✖ 2456
+letter ✗ 24
+letter ❧ 156
+letter כּ 13
+letter ֬ 46-24
+letter ֪ 46-13456
+letter ֩ 46-12345
+letter ֨ 46-1234
+letter ֧ 46-145
+letter ֦ 46-13
+letter ֥ 46-134
+letter ֤ 46-125
+letter ֣ 46-136
+letter ֢ 46-23
+letter ֡ 46-236
+letter ֠ 46-2356
+letter ֟ 46-1345
+letter ֞ 46-12
+letter ֝ 46-16
+letter ֜ 46-1
+letter ֛ 46-1236
+letter ֚ 46-245
+letter ֫ 46-135
+letter ֭ 46-24
+letter ײ 245-245
+letter ֮ 46-1356
+letter ֲ 25
+letter ֳ 345
+letter ִ 24
+letter ֵ 34
+letter ֶ 15
+letter ַ 14
+letter ָ 126
+letter ֹ 246
+letter ֺ 5-246
+letter ֻ 346
+letter ּ 5
+letter ֽ 4
+letter ־ 36
+letter ֿ 2
+letter ׀ 456
+letter ׁ 4
+letter ֯ 456-134
+letter כ 16
+letter ְ 3
+letter ר 1235
+letter ץ 2346
+letter בּ 12
+letter ב 1236
+letter מ 134
+letter ק 12345
+letter ג 1245
+letter ד 145
+letter צ 2346
+letter פ 1234
+letter א 1
+letter ֱ 26
+letter ה 125
+letter ף 124
+letter ע 1246
+letter ס 234
+letter נ 1345
+letter ו 2456
+letter ש 146
+letter ״ 456
+letter ן 1345
+letter װ 2456-2456-0
```

Comment:
is this double vov supposed to have a space after it? that seems like (it could be) a mistake. This looks like the double-vov character used in Yiddish - does its inclusion in this table mean that there's a biblical Hebrew use for the double vov? I am so curious!